[2023-08-31 21:34:25,216][ignite.engine.engine.Engine][INFO] - Engine run starting with max_epochs=1.
[2023-08-31 21:34:27,517][ignite.engine.engine.Engine][ERROR] - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 4.69 GiB (GPU 0; 23.69 GiB total capacity; 15.75 GiB already allocated; 3.39 GiB free; 18.01 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
[2023-08-31 21:34:27,571][ignite.engine.engine.Engine][ERROR] - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 4.69 GiB (GPU 0; 23.69 GiB total capacity; 15.75 GiB already allocated; 3.39 GiB free; 18.01 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
